### PR TITLE
[202012]Fixing db_migrator for Feature table 

### PR DESCRIPTION
**Pull request by dgsudharsan**
This is a cherrypick of https://github.com/Azure/sonic-utilities/pull/1674 for 202012 since the master checks are failing and we need to unblock the 202012 branch

#### What I did
Fixing db_migrator.py for FEATURE table. In case of version_unknown, all versions' migrator APIs would be invoked which will also invoke the feature_table migration. Without the field check for 'status' the current logic will set it 'disabled' and will set 'state' field too to be 'disabled' which will in turn disable all the features bringing down the system.

#### How I did it
Added check to migrate only if 'status' field is present.

#### How to verify it
Install the image through onie and confirm if services are up

### CODE REVIEW

File: scripts/db_migrator.py [hunk 1]
```diff
@@ -181,9 +181,11 @@ def migrate_feature_table(self):
...
         feature_table = self.configDB.get_table('FEATURE')
         for feature, config in feature_table.items():
-            state = config.pop('status', 'disabled')
-            config['state'] = state
-            self.configDB.set_entry('FEATURE', feature, config)
+            state = config.get('status')
+            if state is not None:
+                config['state'] = state
+                config.pop('status')
+                self.configDB.set_entry('FEATURE', feature, config)
```
Comment (yxieca) [inline scripts/db_migrator.py:188]: The behavior is different, the removed code default status to 'disabled'. The added change didn't do that. Is this change intentional? If so, please clarify the intention.

Comment (dgsudharsan) [reply scripts/db_migrator.py:188]: > The behavior is different, the removed code default status to 'disabled'. The added change didn't do that. Is this change intentional? If so, please clarify the intention.

Ying, The db_migrator should modify the field only if present. That's why the if check is added. If the field is not present in FEATURE_TABLE the db_migrator shouldn't assume it to be default disabled and leave it to the specific application which handles it to handle the default.

**Comment by liat-grozovik** [comment]
@yxieca FYI the PR was validated internally and the issue seen on 202012 is now fixed. 
Please let us know if you have any additional concern.

**Review by yxieca — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]
